src/pages/docs/packaging-applications/build-servers/github-actions.md

```diff
@@ -99,6 +97,7 @@ jobs:
 | `OCTOPUS_SPACE`   | The Name of the Space where this command will be executed.                                                                                               |
 | `OCTOPUS_URL`     | The base URL hosting Octopus Deploy (i.e. `https://octopus.example.app`). It is strongly recommended that this value retrieved from a [GitHub secret](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions). |
 | `OCTOPUS_API_KEY` | The API key used to access Octopus Deploy. It is strongly recommended that this value retrieved from a [GitHub secret](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions). |
+| `SERVER`          | The base URL hosting Octopus Deploy (i.e. `https://octopus.example.app`). It is strongly recommended that this value retrieved from a [GitHub secret](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions). |
```

Comment:
I've just moved this one up from the Inputs list as it's actually an Environment Variable in the example.

Reply:
Just want to confirm: Is `SERVER` the right environment variable here? It doesn't sound familiar to me, I _thought_ it was `OCTOPUS_URL` or maybe `OCTOPUS_SERVER`???

Reply:
Ahhh! You're right. It is. I got confused between the 'server' variable and the 'OCTOPUS_URL' env var.